app/controllers/recipes_controller.rb

```diff
@@ -22,6 +27,6 @@ def show
   private
 
   def recipe_params
-    params.require(:recipe).permit(:name, :style, :yeast, :summary, :notes)
+    params.require(:recipe).permit(:name, :style, :grains, :yeast, :summary, :notes)
```

Comment:
See the Strong Parameters section of Rails Guides: http://guides.rubyonrails.org/action_controller_overview.html#strong-parameters

Within that section look for the 2 configuration examples for working with `accepts_nested_attributes_for`. I think that'll get this closer to working as needed.